azure/batch/scripts/predict_on_catalog.py

```diff
@@ -127,7 +127,7 @@ def save_predictions_to_json(predictions: np.ndarray, image_ids: List[str], labe
       'schema': {
         'version': 1,
         'type': 'zooniverse/subject_assistant',
-        'data': { 'subject_id': ['probability_at_least_20pc_featured', [['smooth-or-featured-cd_smooth_prediction'], ['smooth-or-featured-cd_featured-or-disk_prediction'], ['smooth-or-featured-cd_problem_prediction']] ] }
+        'data': { 'subject_id': ['probability_at_least_20pc_featured', ['smooth-or-featured-cd_smooth_prediction', 'smooth-or-featured-cd_featured-or-disk_prediction'], ['smooth-or-featured-cd_problem_prediction'] ] }
```

Comment:
didn't we decide that the non-flattened schema allows for multiple predictions and is what we want here?